build.fsx

```diff
@@ -95,7 +99,14 @@ Target "Build" (fun _ ->
     |> Log "AppBuild-Output: "
 )
 
-// Build lots of packages for differet VM sizes
+let filesForZip =
```

Comment:
Is an easier way to maintain the blacklist of the files that we *don't* want and subtract that from the set of all files?